curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b5a85776edb05d02d23c4.md

```diff
@@ -0,0 +1,126 @@
+---
+id: 646b5a85776edb05d02d23c4
+title: Step 22
+challengeType: 0
+dashedName: step-22
+---
+
+# --description--
+
+You will target the class selector named `dropdown-container` inside the CSS file and add these:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed